docs/email/MX-Backup.md

```diff
@@ -0,0 +1,161 @@
+This is why automated email backups are important to everyone!
+
+And it isn't even that hard to do. Thanks to IMAP being a standart and everyone supporting it as the legacy access to their email servers next to their fancy new http based APIs it is possible for us to use great open source tools like imapsync to quickly create a copy.
+
+## Prerequesities
+In order to setup a backup MX Server you should first consult the guides on how [Getting Started](/docs/gettings-started) how to secure your linode and how to setup a working email server with [mySQL, Dovecot and Postfix](/docs/email/email-with-postfix-dovecot-and-mysql).
+
+### Bonus
+If you want to make your life easier you should go and take a look at how to setup [phpmyadmin](/docs/databases/mysql/mysql-phpmyadmin-ubuntu-14-04) in order to more easily maintain your TLDs and users.
+If you would like to have easier access to the emails, for quick check ups for example, having something like [Roundcube](https://roundcube.net) or [Horde](http://www.horde.org/apps/webmail) installed is also a bonus. It will make everything easier and you don't waste more than 15 minutes on it.
+
+
+## IMAPSYNC
+[imapsync](http://imapsync.lamiral.info) is a command line tool written in Perl by Gilles Lamiral from France. You can buy it off of his [website](http://imapsync.lamiral.info/) and get lifetime instant updates and support (not instant) or you could use the open source version off of Github.com which he updates to the latest version usually a couple days or weeks later. So it is pretty fair use.
+
+
+{: .note}
+>
+> For this guide we will use the Github version. You should really go and buy it though if you really like it. It's a great tool and the maintainer is really nice.
+
+
+### Installing imapsync
+
+We will get a copy of imapsync off of Github with the help of git which comes pre-installed with a lot of Linux distributions nowadays.
+
+	`git clone https://github.com/imapsync/imapsync.git`
+	
+At this point we would just run the programm but it depends on a couple of perl libraries which might have to be installed first. Monsieur Lammiral has a list with a couple of Linux distributions and how to install the dependencies in his Github repo.
+
+{: .note}
+>
+> You should take a look at it if you use something different than Debian/Ubuntu. [Here](https://github.com/imapsync/imapsync/tree/master/INSTALL.d)
+
+	sudo apt-get install \ 
```

Comment:
These are not commands that you can run directly from the terminal. Please explain this section better.